### PR TITLE
Embed DDB bindings to reduce code duplication

### LINKED ISSUE #219: Bring native bindings into Registry
While having a separate nuget package for bindings might seem like a clever choice, it actually hinders development efforts by needing to maintain two separate interfaces (the bindings themselves, plus the adapters/ports in registry).

Just take a look at the adapters code in Registry; it duplicates a lot of model definitions.

We have no known users of the nuget package.

We should make the design easier.

### PR DESCRIPTION
**Pull request by HeDo88TH**
Closes #219

### CODE REVIEW

**Comment by HeDo88TH** [comment]
The tests will not pass until we merge https://github.com/DroneDB/DroneDB/pull/297

File: Registry.Adapters.Ddb/Model/CopyAction.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
+﻿using System;
```
Comment (pierotofy) [inline Registry.Adapters.Ddb/Model/CopyAction.cs:1]: Pretty sure I've removed CopyAction (how did it end up here?)

**Review by pierotofy — APPROVED** [review]
One file got included by mistake, but otherwise looks good. Merge when you're ready.

File: Registry.Adapters.Ddb/Model/CopyAction.cs (hunk 1 above)
Comment (HeDo88TH) [reply Registry.Adapters.Ddb/Model/CopyAction.cs:1]: Maybe I am wrong but I think you did not: https://github.com/DroneDB/DDB.Bindings/blob/main/DDB.Bindings/Model/CopyAction.cs

Comment (HeDo88TH) [reply Registry.Adapters.Ddb/Model/CopyAction.cs:1]: 👌

**Pull request merged**